src/Symfony/Component/HttpKernel/EventListener/RouterListener.php

```diff
@@ -140,7 +140,8 @@ public function onKernelRequest(GetResponseEvent $event)
             }
 
             if (null !== $this->logger) {
-                $this->logger->info(sprintf('Matched route "%s"', $parameters['_route']), $parameters);
+                $logContext = $parameters + array('request-uri' => $request->getUri());
```

Comment:
please distinguish route params and request uri, i.e. array('route-params' => $parameters, 'request-uri' => $request->getUri())`